frontend/tests/unit/store/objectStore.spec.ts

```diff
@@ -70,7 +70,7 @@ describe('Object Store', () => {
 
   let createObjectSpy: SpyInstance;
   let deleteObjectSpy: SpyInstance;
-  let fetchObjectsSpy: SpyInstance;
+  // let fetchObjectsSpy: SpyInstance;
```

Comment:
Not a big concern, just wondering why fetchObjectsSpy are commented and not removed?